src/components/AppMap.vue

```diff
@@ -65,8 +66,89 @@
           <code>bonus</code>: Minimum bonus modifier tier (0, 1, 2 or 3)<br>
           <code>region</code>: Tower region name (e.g. Tabantha, Lanayru, Hateno) <br>
           <code>fieldarea</code>: Field Map area (0 - 93)<br>
-          <code>lotm</code>: Spawns with Lord of the Mountain (1 or 0)<br>
-        </p>
+          <code>lotm</code>: Spawns with Lord of the Mountain (1 or 0) <br>
+          <code>location</code>: items possibly in, beneath or above something
```

Comment:
nit:

```suggestion
          <code>location</code>: Item possibly in, beneath or above something
```

(for consistency)